src/InfluxDB/Database.php

```diff
@@ -47,14 +76,15 @@ class Database
      *
      * @throws \InvalidArgumentException
      */
-    public function __construct($name, Client $client)
+    public function __construct($name, Client $client, InfluxDBPrecision $precision = NULL)
```

Comment:
`@param` needs to be added, and the parameter type should be nullable?

Reply:
No, we want the default to be null. But if you pass it it should be a precision class.

Reply:
No, we want the default to be null. But if you pass it it should be a precision class.